austrakka/components/field/__init__.py

```diff
@@ -38,16 +38,19 @@ def field_list(out_format: str):
               help="Default order in which this column will be sorted in tables relative to other '"
                    "fields. If no value is specifed, the column will be placed after ordered "
                     "columns.")
+@click.option('--private', 'private', is_flag=True, default=False,
```

Comment:
Instead of @click.option, use @create_option (which builds off our option class)